Makefile

```diff
@@ -271,9 +271,12 @@ build: $(DEV_BUILDS)
 # release/build builds a release build for each platform.
 release/build: $(RELEASE_BUILDS)
 
+release/zips: $(RELEASE_ZIPS)
+	@for Z in $^; do echo $$Z; done
+
 # release builds zip-packaged builds for each platform.
 release: $(RELEASE_ZIPS)
-	@for Z in $^; do echo $$Z; done
+	@./dev/release/create $(RELEASE_ZIPS) || exit 1
```

Comment:
Previously release just listed the zips, now it creates a release including them as assets.